src/npm/hook.js

```diff
@@ -131,20 +129,13 @@ const send = (data, port) => {
   )
 }
 
-const getRequiredMeasures = (components, measures) => {
-  let requiredMeasures = []
-
-  if (!Array.isArray(components)) {
-    components = [components]
-  }
-
-  measures.forEach(measure => {
-    if (components.includes(measure.componentName)) {
-      requiredMeasures.push(measure)
-    }
-  })
-
-  return requiredMeasures
-}
+const getMeasuresByComponentName = (componentNames, measures) =>
+  measures.reduce(
```

Comment:
I realised that this is `filter`. update coming (eventually)

Reply:
Nice!